File: Lista2/Python/Lsala2/Ex5.py
a = int(input('Digite o primeiro número: '))
b = int(input('Digite o segundo número: '))
c = int(input('Digite o terceiro número: '))
if a<b and b<c:
    print(f'Os valores em ordem crescente são {a}, {b}, {c}')
elif a<c and c<b:
    print(f'Os valores em ordem crescente são {a}, {c}, {b}')
elif b<a and a<c:
    print(f'Os valores em ordem crescente são {b}, {a}, {c}')
elif b<c and c<a:
    print(f'Os valores em ordem crescente são {b}, {c}, {a}')
elif c<a and a<b:
    print(f'Os valores em ordem crescente são {c}, {a}, {b}')
elif c<b and b<a:
    print(f'Os valores em ordem crescente são {c}, {b}, {a}')
else:
    print('Os valores são iguais')

File: Lista2/C/Lsala2/Ex5.c
	#include <stdio.h> 

	int main()
	{
		int a, b, c;
		printf("Primeiro valor: ");
		scanf("%i", &a);
		printf("Segundo valor: ");
		scanf("%i", &b);
		printf("Terceiro valor: ");
		scanf("%i", &c);
		if(a<b && b<c){
			printf("Os valores em ordem crescente sao %i %i %i", a, b, c);
		}
		else if(a<c && c<b){
			printf("Os valores em ordem crescente sao %i %i %i", a, c, b);
		}
		else if(b<a && a<c){
			printf("Os valores em ordem crescente sao %i %i %i", b, a, c);
		}
		else if(b<c && c<a){
			printf("Os valores em ordem crescente sao %i %i %i", b, c, a);
		}
		else if(c<a && a<b){
			printf("Os valores em ordem crescente sao %i %i %i", c, a, b);
		}
		else if(c<b && b<a){
			printf("Os valores em ordem crescente sao %i %i %i", c, b, a);
		}
		else{
			printf("O valores sao iguais");
		}
	}

